ci/build.yml

```diff
@@ -34,6 +34,38 @@ build and push test image:
     - docker push $CONTAINER_NAME:$CI_COMMIT_SHA
     - docker push $CONTAINER_NAME:test
 
+# trezor-user-env arm docker images build
+.build_arm: &build_arm
+  variables:
+    CONTAINER_NAME_ARM: "$CI_REGISTRY/satoshilabs/trezor/trezor-user-env/trezor-user-env-arm"
+  before_script:
+    - docker login $CI_REGISTRY -u $CI_REGISTRY_USER -p $CI_REGISTRY_PASSWORD
+  tags:
+    - darwin_arm
+
+build and push arm image:
+  stage: build
+  <<: *build_arm
+  only:
+    - schedules  # nightly build
+    - master
+    - /^run\//
+  script:
+    - docker build --tag $CONTAINER_NAME_ARM:latest  -f docker/arm/Dockerfile .
+    - docker push $CONTAINER_NAME_ARM:latest
+
+build and push arm test image:
+  stage: build
+  <<: *build
+  when: manual
+  except:
+    - schedules
+    - master
+    - /^run\//
+  script:
+    - docker build  --tag $CONTAINER_NAME_ARM:CI_COMMIT_BRANCH-test -f docker/arm/Dockerfile .
+    - docker push $CONTAINER_NAME_ARM:CI_COMMIT_BRANCH-test
```

Comment:
Let's remove this, we are reworking this in https://github.com/trezor/trezor-user-env/pull/127 anyway.

Reply:
You meant `<<: *build_arm` anyway :).

Let's remove the anchor altogether then.

Reply:
removed it in c0220b4